heaps/min_heap.py

```diff
@@ -44,10 +61,10 @@ def __str__(self):
 
     def empty(self):
         """ This method returns true if the heap is empty
-            Time complexity: ?
-            Space complexity: ?
+            Time complexity: O(1)
+            Space complexity: O(1)
         """
-        pass
+        return len(self.store) == 0
 
 
     def heap_up(self, index):
```

Comment:
👀 Time complexity would be O (log n) since in the worst case you would swap nodes once per level. Space complexity would also be O(log n) because of the recursive call stack.